### ISSUE TITLE
unreachable/invalid links in discovery metadata generated by wis2box

### ISSUE THREAD

**Issue by maaikelimper** [wmo-im/wis2box#620]
The metadata record generated by wis2box current includes links with type='OAFeat' and type='MQTT' that look as follows:
```
{
"href": "http://localhost:4480/oapi/collections/urn:x-wmo:md:cd-brazza_met_centre:surface-weather-observations",
"type": "OAFeat",
"rel": "collection",
"title": "urn:x-wmo:md:cd-brazza_met_centre:surface-weather-observations"
},
{
"href": "mqtt://everyone:everyone@mosquitto:1883",
"type": "MQTT",
"rel": "data",
"title": "cd-brazza_met_centre.data.core.weather.surface-based-observations.synop",
"channel": "origin/a/wis2/cd-brazza_met_centre/data/core/weather/surface-based-observations/synop"
},
```

The OAFeat collection might not exists and/or is empty depending on how the data-mappings is configured for the dataset. It should be included when the user has enable API-access for their dataset.

_mosquitto_ refers to the hostname within the docker-stack and is unreachable

**Comment by tomkralidis** [comment]
Can you verify that the outgoing WCMP2 document has these links?  The links should not be emitted as part of the WIS2 publication.

**Comment by tomkralidis** [comment]
2024-02-13:
- needs further testing (@tomkralidis / @maaikelimper)

**Comment by tomkralidis** [comment]
To discuss on 2024-03-08

**Comment by tomkralidis** [comment]
Implemented in #642
